x-pack/test/functional/apps/lens/group4/dashboard.ts

```diff
@@ -251,8 +251,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     it('save lens panel to embeddable library', async () => {
       const originalPanel = await testSubjects.find('embeddablePanelHeading-lnsPieVis');
       await panelActions.saveToLibrary('lnsPieVis - copy', originalPanel);
-      await testSubjects.click('confirmSaveSavedObjectButton');
-      await testSubjects.existOrFail('addPanelToLibrarySuccess');
```

Comment:
Now that we are waiting for the object to be added to the library **as part of** the `saveToLibrary` method, this was causing a failure because the `'confirmSaveSavedObjectButton'` was no longer present on the screen.